pkg/webhooks/validation/controller/controller.go

```diff
@@ -233,13 +247,16 @@ func newController(
 		return nil, err
 	}
 
+	dynamicResourceInterface := dface.Resource(istioGatewayGVK).Namespace(o.WatchedNamespace)
```

Comment:
We can use the istio client-go generated api

Reply:
We could. This PR was going to be cherry-picked into 1.5 I was trying to avoid adding dependencies last minute. But maybe that isn't a concern in this context.

Reply:
Is it even appropriate for istio to depend on client-go? I am not sure the desired dependency tree shape. I guess k8s does it